src/x11_init.c

```diff
@@ -458,6 +458,16 @@ static void detectEWMH(void)
     XFree(supportedAtoms);
 }
 
+XRRScreenResources* _glfwXRRGetScreenResources()
+{
+#if (RANDR_MAJOR > 1 || (RANDR_MAJOR == 1 && RANDR_MINOR >= 3))
```

Comment:
Version 1.3 is already required for RandR to be used (see `initExtensions`) so this function should be replaced with direct calls to `XRRGetScreenResourcesCurrent`.